starter-code/models/auth.js

```diff
@@ -0,0 +1,16 @@
+const mongoose = require("mongoose");
+const Schema = mongoose.Schema;
+
+const AuthSchema = new Schema({
+  username: {
+    type: String,
+    required: true,
+    unique: true
+  },
+  password: {
+    type: String,
+    required: true
+  }
+});
+
+module.exports = mongoose.model("Auth", AuthSchema);
```

Comment:
It probably makes more sense to create a collection of `users` instead of `auths`